spec/features/users/forgot_password_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+RSpec.describe ForgotPassword do
+  let(:forgot_password) { described_class.new }
+
+  before { forgot_password.load }
+
+  it 'show form forgot password' do
+    expect(forgot_password).to have_cancel_link
+    expect(forgot_password).to have_forgot_title
+    expect(forgot_password).to have_email_button
+    expect(forgot_password).to have_user_email
+    expect(forgot_password).to have_email_instriction
+  end
+
+  describe 'Click to email in button' do
+    it 'Empty email and password' do
```

Comment:
Rename this test according to logic of him
You test that alert message is display, so:
```suggestion
    it 'displays alert message' do
```